src/google/protobuf/json/internal/unparser.cc

```diff
@@ -831,22 +831,40 @@ absl::Status WriteMessage(JsonWriter& writer, const Msg<Traits>& msg,
 }
 }  // namespace
 
-absl::Status MessageToJsonString(const Message& message, std::string* output,
+absl::Status MessageToJsonStream(const Message& message, 
+                                 io::ZeroCopyOutputStream* json_output,
                                  json_internal::WriterOptions options) {
+  // NOTE: Most of the contortions in this function are to allow for capture of
```

Comment:
Could you update this comment to reflect that we're only capturing the output (not input) in ABSL_DLOG mode?

Reply:
done